File: chpt 3 assign translated to java/MilesPerGallon.java
import java.util.Scanner;

public class MilesPerGallon {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        double totalMiles = 0;
        double totalGallons = 0;

        while (true) {
            System.out.print("Enter gallons used (-1 to end): ");
            double gallons = scanner.nextDouble();
            if (gallons == -1) {
                break;
            }

            System.out.print("Enter miles driven: ");
            double miles = scanner.nextDouble();

            double mpg = miles / gallons;
            System.out.printf("The miles/gallon for this tank was %.6f%n", mpg);

            totalMiles += miles;
            totalGallons += gallons;
        }

        double combinedMpg = totalMiles / totalGallons;
        System.out.printf("The overall average miles/gallon was %.6f%n", combinedMpg);
    }
}

File: python chpt 3 assignments/milespergallon.py
print("WELCOME TO JOHNSON WORLD")

total_miles = 0
total_gallons = 0

while True:
    gallons = float(input("Enter gallons used (-1 to end): "))
    if gallons == -1:
        break

    miles = float(input("Enter miles driven: "))

    mpg = miles / gallons
    print(f"The miles/gallon for this tank was {mpg:.6f}")

    total_miles += miles
    total_gallons += gallons

combined_mpg = total_miles / total_gallons
print(f"The overall average miles/gallon was {combined_mpg:.6f}")



